### PR TITLE
Renamed 'release' references(vars, classes and logs) to 'pegout'

### PR DESCRIPTION
**Pull request by nathanieliov**

### CODE REVIEW

File: src/main/java/co/rsk/federate/signing/hsm/message/PegoutCreationInformation.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    private final Keccak256 pegoutCreationRskTxHash;
+    private final Keccak256 pegoutConfirmationRskTxHash;
+    private final BtcTransaction pegoutBtcTx;
+
+
+    /**
+     *
+     * @param pegoutCreationRskBlock                 The rsk block where the BTC transaction was created
+     * @param transactionReceipt    The rsk transaction receipt where the btc transaction was created
+     * @param pegoutCreationRskTxHash      The rsk transaction hash where the pegout was requested
```
Comment (kelvinator07) [inline src/main/java/co/rsk/federate/signing/hsm/message/PegoutCreationInformation.java:20]:
```suggestion
     * @param pegoutCreationRskTxHash      The rsk transaction hash where the pegout was created
```

File: src/main/java/co/rsk/federate/signing/hsm/message/PegoutCreationInformation.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        BtcTransaction pegoutBtcTx
+    ) {
+        this(pegoutCreationRskBlock, transactionReceipt, pegoutCreationRskTxHash, pegoutBtcTx, pegoutCreationRskTxHash);
+    }
+
+    /**
+     *
+     * @param pegoutCreationRskBlock                 The rsk block where the BTC transaction was created
+     * @param transactionReceipt    The rsk transaction receipt where the btc transaction was created
+     * @param pegoutCreationRskTxHash      The rsk transaction hash where the pegout was requested
```
Comment (kelvinator07) [inline src/main/java/co/rsk/federate/signing/hsm/message/PegoutCreationInformation.java:36]:
```suggestion
     * @param pegoutCreationRskTxHash      The rsk transaction hash where the pegout was created
```

File: src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java [hunk 3]
```diff
@@ -39,103 +39,103 @@ public ReleaseCreationInformationGetter(
...
-    public ReleaseCreationInformation getTxInfoToSign(
+    public PegoutCreationInformation getPegoutCreationInformationToSign(
         int version,
-        Keccak256 rskTxHash,
-        BtcTransaction btcTransaction,
-        Keccak256 informingRskTxHash
+        Keccak256 pegoutCreationRskTxHash,
+        BtcTransaction pegoutBtcTx,
+        Keccak256 pegoutConfirmationRskTxHash
     ) throws HSMReleaseCreationInformationException {
```
Comment (kelvinator07) [inline src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java:55]:
```suggestion
    ) throws HSMPegoutCreationInformationException {
```

File: src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java [hunk 4]
```diff
@@ -148,8 +148,8 @@ private ReleaseCreationInformation searchEventInFollowingBlocks(
             TransactionInfo transactionInfo = receiptStore.getInMainChain(transaction.getHash().getBytes(), blockStore).orElse(null);
             TransactionReceipt transactionReceipt = transactionInfo.getReceipt();
             transactionReceipt.setTransaction(transaction);
-            Optional<ReleaseCreationInformation> optionalReleaseCreationInformation =
-                getInformationFromEvent(block, transactionReceipt, btcTransaction, rskTxHash, informingRskTxHash);
+            Optional<PegoutCreationInformation> optionalReleaseCreationInformation =
```
Comment (kelvinator07) [inline src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java:151]:
```suggestion
            Optional<PegoutCreationInformation> optionalPegoutCreationInformation =
```

File: src/main/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcer.java
Comment (kelvinator07) [inline src/main/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcer.java:1]: ReleaseRequirementsEnforcer -> PegoutSigningRequirementsEnforcer

File: src/main/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcer.java [hunk 5]
```diff
@@ -13,7 +13,7 @@ public ReleaseRequirementsEnforcer(AncestorBlockUpdater ancestorBlockUpdater) {
         this.ancestorBlockUpdater = ancestorBlockUpdater;
     }
 
-    public void enforce(int version, ReleaseCreationInformation releaseCreationInformation)
+    public void enforce(int version, PegoutCreationInformation pegoutCreationInformation)
         throws ReleaseRequirementsEnforcerException {
```
Comment (kelvinator07) [inline src/main/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcer.java:17]:
```suggestion
        throws PegoutSigningRequirementsEnforcerException {
```

File: src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java [hunk 6]
```diff
@@ -214,7 +214,7 @@ public void processReleases_ok() throws Exception {
         SignerMessageBuilder messageBuilder = new SignerMessageBuilderVersion1(releaseTx);
```
Comment (kelvinator07) [inline src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java:214]:
```suggestion
        SignerMessageBuilder messageBuilder = new SignerMessageBuilderVersion1(pegoutBtcTx);
```

File: src/test/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcerTest.java [hunk 7]
```diff
@@ -19,7 +18,7 @@ public void enforce_does_nothing_if_version_one()
         AncestorBlockUpdater ancestorBlockUpdater = mock(AncestorBlockUpdater.class);
         ReleaseRequirementsEnforcer enforcer = new ReleaseRequirementsEnforcer(ancestorBlockUpdater);
```
Comment (kelvinator07) [inline src/test/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcerTest.java:19]:
```suggestion
        PegoutSigningRequirementsEnforcer enforcer = new ReleaseRequirementsEnforcer(ancestorBlockUpdater);
```

File: src/main/java/co/rsk/federate/signing/hsm/message/PegoutCreationInformation.java (hunk 1 above)
Comment (nathanieliov) [reply src/main/java/co/rsk/federate/signing/hsm/message/PegoutCreationInformation.java]: Done

File: src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java (hunk 3 above)
Comment (nathanieliov) [reply src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java]: Done

File: src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java (hunk 4 above)
Comment (nathanieliov) [reply src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetter.java]: Done

File: src/main/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcer.java
Comment (nathanieliov) [reply src/main/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcer.java:1]: Done

File: src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java (hunk 6 above)
Comment (nathanieliov) [reply src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java]: Done

File: src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java [hunk 8]
```diff
@@ -700,7 +700,7 @@ public void validateTxCanBeSigned_federationCantSign() throws Exception {
         );
 
         // Act
-        client.validateTxCanBeSigned(releaseTx);
+        client.validateConfirmedPegoutCanBeSigned(releaseTx);
```
[the remark this answers is not in the document]
Comment (nathanieliov) [reply src/test/java/co/rsk/federate/btcreleaseclient/BtcReleaseClientTest.java]: Done

File: src/test/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcerTest.java (hunk 7 above)
Comment (nathanieliov) [reply src/test/java/co/rsk/federate/signing/hsm/requirements/ReleaseRequirementsEnforcerTest.java]: Done

**Review by kelvinator07 — APPROVED** [review]

**Pull request merged**